LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/import/animate-elem-41-t-manual-expected.txt

```diff
@@ -100,7 +100,7 @@ layer at (0,0) size 800x600
             RenderSVGContainer {g} at (455,150) size 38x36 [transform={m=((1.00,0.00)(0.00,1.00)) t=(110.00,0.00)}]
               RenderSVGPath {polyline} at (455,150) size 38x36 [stroke={[type=SOLID] [color=#CC0066]}] [fill={[type=SOLID] [color=#CC0066]}] [points="20 10 0 10 15 20 10 2 5 20 20 10"]
             RenderSVGContainer {a} at (638,150) size 38x36 [transform={m=((1.00,0.00)(0.00,1.00)) t=(220.00,0.00)}]
-              RenderSVGPath {polyline} at (638,150) size 38x36 [stroke={[type=SOLID] [color=#CC0066]}] [fill={[type=SOLID] [color=#000000]}] [points="20 10 0 10 15 20 10 2 5 20 20 10"]
+              RenderSVGPath {polyline} at (638,150) size 38x36 [stroke={[type=SOLID] [color=#CC0066]}] [fill={[type=SOLID] [color=#CC0066]}] [points="20 10 0 10 15 20 10 2 5 20 20 10"]
```

Comment:
Did you confirm these are progressions?

Reply:
Yes, they are fixing the same bug indeed. 

```
<g id="setEleven" color="rgb(204,0,102)" transform="translate(0,200)">
...
     <a xlink:href="" transform="translate(220, 0)" fill="currentColor">
```

Reply:
The testcase in question isn't using currentColor, but it is a similar bug with applying a `visitedLinkFillPaintColor` that is uninitialized. Here's a reduced testcase:
```
<svg width="100" height="100" fill="red" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <g fill="green">
    <a xlink:href="">
      <rect width="100" height="100"></rect>
    </a>
  </g>
</svg>
```

We should add a WPT for this.